### DISCUSSION TITLE
Password for Terminal Login

### DISCUSSION THREAD

**Discussion by adnan29979** [Ylianst/MeshCentral#5910] [Q&A] [answered]
Hello. 
Is it possible to configure MeshCentral so that accessing a device's terminal requires entering the device's password?

**Comment by si458** [comment, marked as answer]
No and yes
No the idea is to just jump straight into a terminal session
Yes you can right click the connect button and select the login prompt option 
So you can then enter a completely different user/pass

You would need to open a github issue enhancement request to add a feature like set the default terminal option on connect

**Reply by adnan29979** [reply]
Oh okay, got it. Thanks for the insight.
